src/context.cc

```diff
@@ -211,6 +221,11 @@ void Context::InitSim(SimInfo si) {
       ->AddVal("DurationSecs", static_cast<int>(si.dt))
       ->Record();
 
+  NewDatum("RNGInfo")
+      ->AddVal("Seed", static_cast<int>(si.seed))
+      ->AddVal("Stride", static_cast<int>(si.stride))
+      ->Record();
+
```

Comment:
I don't think these need to be added here as well as in the "Info" block above.  I'm not sure which is better, but I'm guessing the one above will be good enough.

You may still need the `static_cast` though.

Reply:
Wasn't sure if you had an opinion about adding to Info vs their own data table. If you think the former is fine, I'll just do that